src/main/java/bio/terra/common/flagsmith/FlagsmithClient.java

```diff
@@ -0,0 +1,101 @@
+package bio.terra.common.flagsmith;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import bio.terra.common.exception.BadRequestException;
+import com.flagsmith.exceptions.FlagsmithClientError;
+import com.flagsmith.models.Flags;
+import java.util.Map;
+import java.util.Optional;
+import javax.annotation.Nullable;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+
+public class FlagsmithClient {
+  private static final Logger LOGGER = getLogger(FlagsmithClient.class);
+  private final com.flagsmith.FlagsmithClient flagsmith;
```

Comment:
why not just import `FlagsmithClient`?

Reply:
this library wraps the identity traits we want to query against, example - email_address. 
i would anticipate we use more traits in the future. 

adding this wrapper will reduce the maintenance overhead (avoids repeating code in multiple repos)

Reply:
Ah to clarify, I meant why not just import `FlagsmithClient` above and have `private final FlagsmithClient flagsmith;` here

Reply:
Ah. gotcha. readability. just so we dont confuse it to a reference to the client class we are creating